shipctl/x86_64/Ubuntu_16.04/shippable_bump_version

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+shippable_bump_verison() {
+  local version_to_bump=$1
+  local action=$2
+  IFS='.' read -ra versionParts <<< "$version_to_bump"
+  IFS=' '
+  local replace=""
+  local numRegex='^[0-9]+$'
+  local major=$(echo ${versionParts[0]} | sed "s/v/${replace}/")
+  local minor=${versionParts[1]}
+  local patch=${versionParts[2]}
+  if ! [[ $action == "major" || $action == "minor" || $action == "patch" ]]; then
+    echo "error: Invalid action given in the argument." >&2; exit 1
+  fi
+  if ! [[ $major =~ $numRegex && $minor =~ $numRegex && $patch =~ $numRegex ]] ; then
+    echo "error: Invalid semantics given in the argument." >&2; exit 1
+  fi
+  if [[ ${versionParts[0]} == $major ]]; then
```

Comment:
I think we can completely eliminate `appendV` if we just move this check below where we need to decide if we want to append "v" or not.